traffic_portal/app/src/common/api/TenantService.js

```diff
@@ -44,11 +44,11 @@ var TenantService = function($http, ENV, messageModel) {
     this.createTenant = function(tenant) {
         return $http.post(ENV.api['root'] + 'tenants', tenant).then(
             function(result) {
-                messageModel.setMessages([ { level: 'success', text: 'Tenant created' } ], true);
+                messageModel.setMessages([ { level: 'success', text: 'Tenant created' } ], false);
```

Comment:
actually, this one is `true` because successfully creating a tenant redirects you to the tenants list.